cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/AbstractTicketDelegator.java

```diff
@@ -73,7 +75,7 @@ public final TicketGrantingTicket getGrantingTicket() {
     }
 
     @Override
-    public final long getCreationTime() {
+    public final ZonedDateTime getCreationTime() {
```

Comment:
it looks like Hibernate 5.0 will support this type:

https://hibernate.atlassian.net/browse/HHH-8844